cmd/test_email/main.go

```diff
@@ -25,6 +25,8 @@ func main() {
 	sendPlanDiscussionCreatedTestEmail(emailService, templateService, addressBook)
 	sendPlanDiscussionTaggedUserTestEmail(emailService, templateService, addressBook)
 	sendPlanDiscussionTaggedSolutionTestEmail(emailService, templateService, addressBook)
+	//DiscussionReply email
```

Comment:
Comment feels redundant

Reply:
perhaps, I wanted to show that this is a section that we can add future discussion reply emails, vs an initial discussion email. I think it's probably extra verbose, but I'd prefer to keep it for this auxiliary package.